plugins/zfs/zfs_count

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#%# family=auto
+#%# capabilities=autoconf
+
+# Michael Grote
```

Comment:
Please add a *perlpod* header.

Maybe you can describe the difference between this plugin and the other existing zfs-related plugins?

Reply:
added